checkov/secrets/runner.py

```diff
@@ -212,7 +216,10 @@ def run(
                     continue
                 result: _CheckResult = {'result': CheckResult.FAILED}
                 try:
-                    line_text = linecache.getline(secret.filename, secret.line_number)
+                    if runner_filter.enable_git_history_secret_scan and code_line is not None:
+                        line_text = code_line or ''
```

Comment:
@omryMen Why `or ''`? what are we protecting against?

Reply:
don't want None

Reply:
super defensive :)